app/controllers/parks_controller.rb

```diff
@@ -7,4 +7,31 @@ def index
   def show
     @park = Park.find(params[:id])
   end
+
+  def new
+    @park = Park.new
+  end
+
+  def create
+    @park = Park.new(park_params)
+    if @park.save
+      flash[:success] = "Park successfully created!"
+      redirect_to root_path
+    else
+      flash[:notice] = @park.errors.full_messages
+      render 'new'
+    end
+  end
+
+  def park_params
+    params.require(:park).permit(
+    :title,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.